pygmt/src/histogram.py

```diff
@@ -10,16 +10,29 @@
     A="horizontal",
     B="frame",
     C="cmap",
+    D="annotate",
+    E="barwidth",
+    F="center",
     G="fill",
     J="projection",
+    L="extreme",
+    N="normal",
+    Q="cumulative",
     R="region",
+    S="step",
```

Comment:
GMT.jl uses `stairs`, but I don't know which one is more accurate.

Reply:
I switched it to `stairs` in a0d64f1 (and then said it was `steps` in the commit message :facepalm: )